File: app/src/main/java/com/example/odisea/data/Lugar.java
package com.example.odisea.data;

public class Lugar {
    private String id;
    private String nombre;
    private String descripcion;
    private String ubicacion;
    private float calificacion;
    private String imagenUrl;
    private String tipoEstablecimiento; // "hotel", "spa", "restaurante", "pista"

    public Lugar(String id, String nombre, String descripcion, String ubicacion, float calificacion, String imagenUrl, String tipoEstablecimiento) {
        this.id = id;
        this.nombre = nombre;
        this.descripcion = descripcion;
        this.ubicacion = ubicacion;
        this.calificacion = calificacion;
        this.imagenUrl = imagenUrl;
        this.tipoEstablecimiento = tipoEstablecimiento;
    }

    public String getId() { return id; }
    public String getNombre() { return nombre; }
    public String getDescripcion() { return descripcion; }
    public String getUbicacion() { return ubicacion; }
    public float getCalificacion() { return calificacion; }
    public String getImagenUrl() { return imagenUrl; }
    public String getTipoEstablecimiento() { return tipoEstablecimiento; }
}

File: app/src/main/java/com/example/odisea/data/Lugar.kt
package com.example.odisea.data

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Lugar(
    val id: Int,
    val nombre: String,
    val ubicacion: String?,
    val valoracion: Float?,
    val descripcion: String?,
    @SerializedName("img_url")
    val imagenUrl: String? = null,
    val detalles: String? = null,
    @SerializedName("tipo_establecimiento")
    val tipoEstablecimiento: String?,
    var esFavorito: Boolean = false
) : Parcelable
